src/util/metadata/vo_bit.rs

```diff
@@ -0,0 +1,97 @@
+//! Valid-object bit (VO-bit)
```

Comment:
I did not find anywhere else we define what the valid object bit is. Probably we can describe valid object bit here. We should make it clear that valid object bit means we have a valid object here: it is an object, it has been allocated, and it hasn't been reclaimed, etc.

Reply:
I added module-level doc comment to describe what valid object bit is and what it is intended for.